Refit/RequestBuilderImplementation.cs

```diff
@@ -330,7 +351,18 @@ void addMultipartItem(MultipartFormDataContent multiPartContent, string itemName
             var taskFunc = buildCancellableTaskFuncForMethod<T>(restMethod);
 
             return (client, paramList) => 
-                new TaskToObservable<T>(ct => taskFunc(client, ct, paramList));
+                new TaskToObservable<T>(ct =>
+                                        {
+                                            var methodCt = CancellationToken.None;
```

Comment:
Not sure exactly how @paulcbetts would want this formatted, but this looks like R# formatting. :smiling_imp: